src/compiler/checker.ts

```diff
@@ -2369,7 +2374,8 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         location: Node,
         maybeMissingAwait: boolean,
         message: DiagnosticMessage,
-        ...args: DiagnosticArguments): Diagnostic {
+        ...args: DiagnosticArguments
+    ): Diagnostic {

```

Comment:
I first thought this was a dprint bug, but no, you really can't have a trailing comma on a `...args` param. That's silly.